.github/workflows/run-k8s-tests-on-aks.yaml

```diff
@@ -103,7 +103,12 @@ jobs:
       - name: Deploy Kata
         timeout-minutes: 10
         run: bash tests/integration/kubernetes/gha-run.sh deploy-kata-aks
-      
+
+      - name: Deploy CoCo KBS
+        if: ${{ matrix.vmm == 'qemu' }}
```

Comment:
I wonder if it's worth passing `${{matrix.vmm}}` to the `gha-run.sh` script below so we can expand it to support other hypervisors in future?

Reply:
`${{matrix.vmm}}` is exported in `KATA_HYPERVISOR` that I can read like from `gha-run.sh`

Are you thinking in have a filter similar to https://github.com/kata-containers/kata-containers/pull/9062/commits/4daed2292bb49b7c3d8b301de0ff7f02354e2f9b ?

Reply:
@wainersm How about putting this condition in the shell script so upcoming PRs can easily modify it?

Reply:
Done.